nginx.conf

```diff
@@ -7,8 +7,8 @@ http {
             proxy_pass http://frontend:3000;
         }
         # api
-        location /api/ {
-            rewrite /api/(.*) /$1 break;
+        location /backend/ {
```

Comment:
changed the name of the `/api` location to not overlap with the one `/api` path of nuxt3.